src/main/java/kirby/exceptions/KirbyInvalidCommandException.java

```diff
@@ -0,0 +1,15 @@
+package kirby.exceptions;
+
+/**
+ * KirbyInvalidCommandException class inherits from KirbyException and
+ * is thrown when there is an undefined input command.
+ */
+public class KirbyInvalidCommandException extends KirbyException {
+    /**
+     * Constructor for the class KirbyInvalidCommandException.
+     */
+    public KirbyInvalidCommandException() {
+        super("Aaaaaaah, I don't understand you :(\n" +
+                "Try adding a valid command (todo, deadline, event)");
```

Comment:
Not sure if this is required as it is not explicitly stated, but do you think it would be better if the `+` is at the start of the next line instead of end of the previous line?
```suggestion
        super("Aaaaaaah, I don't understand you :(\n"
                + "Try adding a valid command (todo, deadline, event)");
```

Reply:
Not sure, also can't find any rule about it. Personally, I find the '+' above easier to read but if there's a more correct way, I'll follow it